docs/install.rst

```diff
@@ -72,25 +76,24 @@ and run setup::
     to manually build the Charm++ library (see below).
 
 
-Manually building the Charm++ shared library
---------------------------------------------
+pip (Python 3.7 and older)
```

Comment:
`pip` works for newer Python versions too. Just that `pip3 install charm4py`  will build from source instead of getting prebuilt binaries